src/applications/gi/components/profile/Programs.jsx

```diff
@@ -45,21 +45,18 @@ export class Programs extends React.Component {
         }
       },
 
+      priorityEnrollment: {
```

Comment:
Just curious why you changed this entry's order in the list.

Reply:
Guess I should stick it on the end. The order shouldn't matter.